sechub-developertools/scripts/sechub-api.sh

```diff
@@ -91,14 +103,26 @@ user_signup_decline <new user-id> - Decline registration of <new user-id>
 EOF
 }
 
+
 #################################
 # Generic helper functions
 
+function check_executable_is_installed(){
+    executable="$1"
+    exe_path=`which $executable`
+    if [ ! -x "$exe_path" ] ; then
+        echo "FATAL: Mandatory executable \"$executable\" not found in PATH. Please install."
+        exit 1
+    fi
+}
+
+
 function curl_with_sechub_auth {
   # Don't reveal secrets in the curl process
   printf "user = $SECHUB_USERID:$SECHUB_APITOKEN\n" | curl --config - $CURL_PARAMS "$@"
```

Comment:
```suggestion
  printf "user = $SECHUB_USERID:$SECHUB_APITOKEN\n" | curl --config - "$CURL_PARAMS" "$@"
```

Reply:
The `"` will break the curl command.
Leaving them as they are.